ast/annotations.go

```diff
@@ -324,6 +332,10 @@ func (a *Annotations) toObject() (*Object, *Error) {
 		obj.Insert(StringTerm("title"), StringTerm(a.Title))
 	}
 
+	if a.Entrypoint {
+		obj.Insert(StringTerm("entrypoint"), BooleanTerm(a.Entrypoint))
```

Comment:
[nit] Since we only include this if it's true, we could also use 
```suggestion
		obj.Insert(StringTerm("entrypoint"), BooleanTerm(true))
```

Reply:
Rolled this suggestion into my latest set of changes. :+1: